src/galaxy/config.py

```diff
@@ -83,6 +83,9 @@
         "value not supported for file_upload_method ,switching to default disk method")
     use_s3_to_upload = False
 
+MAIN_API_URL = config.get("API_MONITORING", "MAIN_API_URL", fallback="http://127.0.0.1:8000/v1/")
+ACCESS_TOKEN = config.get("API_MONITORING", "ACCESS_TOKEN", fallback=None)
```

Comment:
May be checking  and raising the error if access_token is not supplied would be fruitful since the monitoring depends on it , API should be able to run without those configuration , you can check if it is supplied or not when endpoint is called !